docs/tutorial.rst

```diff
@@ -232,4 +236,165 @@ Taking a look at the vcf file, we see something like this:
+for this method, we can see that we'll need two inputs: ``sample_sets`` and
+``indexes``.
+Let's think about what these inputs are, and how we can obtain them with 
+Python commands.
+In our case, the sample sets correspond to the lists 
+of sample chromosomes (nodes) from each separate population.
+We can obtain the necessary list of lists like this:
+
+.. code-block:: python
+
+    >>> sample_list = []
+    >>> for pop in range(0, ts.num_populations):
+    ...     sample_list.append(ts.samples(pop).tolist())
+    >>> print(sample_list)
+    [[0, 1, 2, 3], [4, 5, 6, 7], [8, 9, 10, 11], [12, 13, 14, 15]]
+
+Note that the samples with node IDs 0 - 3 are from population 0,
+samples with node IDs 4 - 7 are from population 1 and so on. 
+
+The indexes are the pairs of integer indexes corresponding to the populations
+that we wish to compare.
+We can do this quickly with the ``itertools`` module:
 
+.. code-block:: python
+
+    >>> import itertools
+    >>> inds = itertools.combinations_with_replacement(range(0, ts.num_populations), 2)
+    >>> inds = list(inds)
+    >>> print(inds)
+    [(0, 0), (0, 1), (0, 2), (0, 3), (1, 1), (1, 2), (1, 3), (2, 2), (2, 3),
+     (3, 3)]
+
+We are now ready to calculate the genetic divergences.
+
+.. code-block:: python
+
+    >>> divs = ts.divergence(sample_sets=sample_list, indexes=inds)
+    >>> print(divs)
+    array([0.00035424, 0.0003687 , 0.00036707, 0.0003705 , 0.00026696,
+       0.00029148, 0.00029008, 0.00025767, 0.0002701 , 0.00028184])
+
+---------------------------
+3. Plotting the divergences
+---------------------------
+
+The output lists the divergences of all population pairs that are specified in
+``indexes``, in the same order.
+However, instead of simply printing these values to the console, it might be nicer
+to create a heatmap of the values.
+Here is some (more advanced) code that does this.
+It relies on the ``numpy``, ``seaborn`` and ``matplotlib`` packages.
+
+.. code-block:: python
+    
+    >>> import numpy as np
+    >>> import seaborn
+    >>> import matplotlib.pyplot as plt
+    >>> import matplotlib.ticker as ticker
+    >>> div_matrix = np.zeros((ts.num_populations, ts.num_populations))
+    >>> for pair in range(0, len(inds)):
+    ...     pop0, pop1 = inds[pair]
+    ...     div_matrix[pop0, pop1] = divs[pair]
+    ...     div_matrix[pop1, pop0] = divs[pair]
+    >>> seaborn.heatmap(div_matrix, vmin=0, vmax=0.0005, square=True)
+    >>> ax = plt.subplot()
+    >>> plt.title("Genetic divergence")
+    >>> plt.xlabel("Populations", fontweight="bold")
+    >>> plt.ylabel("Populations", fontweight="bold")
+    >>> ax.set_xticks([0,1,2,3], minor=True)
+    >>> ax.set_xticklabels(['AFR', 'EUR', 'ASI', 'ADM'], minor=False)
+    >>> ax.tick_params(which='minor', length=0)
+    >>> ax.set_yticks([0,1,2,3], minor=True)
+    >>> ax.set_yticklabels(['AFR', 'EUR', 'ASI', 'ADM'], minor=False)
+    >>> ax.tick_params(which='minor', length=0)
+
+.. image:: pics/tute-divergence.png
```

Comment:
Sorry, should have spotted this before now. Stuff like usually goes in ``_static``. Can you move the image in there please?

Reply:
done now ✅